pkgs/top-level/all-packages.nix

```diff
@@ -5549,7 +5549,13 @@ in
   # These are for compatibility and should not be used inside Nixpkgs.
   pythonFull = python.override{x11Support=true;};
   python2Full = python2.override{x11Support=true;};
+  python26Full = python26.override{x11Support=true;};
+  python27Full = python2Full;
```

Comment:
`python27Full = python27.override...`
While python2 points to python27 it could get messy when overriding one before overriding to get the full version.